Resources/Textures/SimpleStation14/Mobs/Silicon/ai.rsi/meta.json

```diff
@@ -0,0 +1,31 @@
+{
+    "version": 1,
+    "size": { "x": 32, "y": 32 },
+    "license": "CC-BY-SA-3.0",
+    "copyright": "",
```

Comment:
🤨

Reply:
honestly, no idea where the sprites came from if router didn't create them, was a long time ago